.github/workflows/ci.yml

```diff
@@ -242,8 +242,8 @@ jobs:
             "3.11",
             "3.12",
             "3.13",
-            "pypy3.9",
```

Comment:
Historically we've been slow to drop support of PyPy versions (typically we match the version of CPython EOL just to keep things simpler). Do you think it's better if we drop PyPy versions eagerly on EOL? Would certainly mean fewer CI runs 😂 

Also, if we want to drop PyPy 3.9 support, I would rather we removed the CI jobs, started preventing compiles (because it's possible we might break it by accident once CI is removed), and added a newsfragment to announce, all in one step. Probably best to split into a separate PR, if we want to do this now.

Reply:
> Do you think it's better if we drop PyPy versions eagerly on EOL?

It is up to you. I don't think we would do another 3.9 release even if a new issue were discovered. I would try to help you solve them though not with a high priority. Have you had many PyPy-specific issues reported? I will revert this change and let you do it at your tempo.